packages/Manager/Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR application
 COPY --from=builder application/dependencies/ ./
 COPY --from=builder application/spring-boot-loader/ ./
 COPY --from=builder application/application/ ./
-ENTRYPOINT ["java", "-XX:MaxRAMPercentage=85", "org.springframework.boot.loader.JarLauncher"]
+ENTRYPOINT ["java", "-XX:MaxRAMPercentage=80", "org.springframework.boot.loader.JarLauncher"]
```

Comment:
When running this with a full load of data there were problems with hitting memory errors, I believe because the JVM was trying to use too much memory from the underlying infra